### ISSUE TITLE
[Bug]: Can't pull docker image due to wrong manifest version

### ISSUE THREAD

**Issue by KebSeb** [timescale/timescaledb#5414]
### What type of bug is this?

Other

### What subsystems and features are affected?

Other

### What happened?

The newer versions (realeased around a month ago going forwards, like `timescale/timescaledb:2.9.2-pg13`) can not be docker pulled, it crashes with
`mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'` on pull.

Our platform is amd64, the os is  'Yoctua', the hardware itself is an industry E22, if you need some more data just let me know.
Docker version 19.03.2-ce, build 6a30dfca03

We had the same problem for our own docker images here aswell, it was caused by docker buildx multi architecture builds. we fixed it by not using it anymore.

a tag that works is:
timescale/timescaledb:2.9.0-pg13-bitnami

best regards

### TimescaleDB version affected

2.9.2 and more

### PostgreSQL version used

13

### What operating system did you use?

Yoctua, amd64

### What installation method did you use?

Docker

### What platform did you run on?

Other

### Relevant log output and stack trace

```bash
docker pull timescale/timescaledb:2.9.2-pg13
2.9.2-pg13: Pulling from timescale/timescaledb
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'
```

### How can we reproduce the bug?

```bash
- be on Yoctua, amd64, Docker version 19.03.2-ce, build 6a30dfca03

docker pull timescale/timescaledb:2.9.2-pg13

Pulling from timescale/timescaledb
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'
```

**Comment by jnidzwetzki** [comment]
Hello @KebSeb,

Thanks for reaching out. You are using Docker `19.03.2`. This Docker version is affected by a bug that prevents the processing of OCI image manifests `application/vnd.oci.image.manifest.v1+json` (see https://github.com/moby/moby/issues/39727 and https://github.com/docker/engine/pull/405). The bug was fixed in Docker `19.03.5` (https://github.com/docker/engine/commit/967aa3a9ef93330f08d19d61e9819426139d4b5e). Please update your Docker installation to at least this version. Afterward, you should be able to pull the image.

If you still encounter issues after the update, please send us the output of `docker version` for further analysis.

**Comment by KebSeb** [comment]
Thank you for your quick reply, we will see if we can update our docker.
As this might or might not happen in the near future i suggest closing this Ticket as resolved and i would reopen/reissue it incase the update doesn't fix the problem
